app/workers/ping_thing_worker.rb

```diff
@@ -0,0 +1,19 @@
+class PingThingWorker
+  include Sidekiq::Worker
+
+  def perform(args = {})
+    PingThingRaw.first(100).each do |raw|
```

Comment:
I'd move the code to separate service so it's reusable and testable without Sidekiq